### PR TITLE
web: add support for removing of external service information from webhook logs.

### PR DESCRIPTION
**Pull request by sashaostrikov**
Also remove external service information from SiteAdminWebhookPage.

Test plan:
Storybook.

## App preview:

- [Web](https://sg-web-ao-ui-webhooks-remove-ext-svc.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | 0.00% (+0.06 kb) | 0.00% (+0.06 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/ao/ui/webhooks/remove-ext-svc-column/compare-report.html;tab=live_object) for a full comparison between the commits d2d9281 and 260561a or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

**Review by mrnugget — APPROVED** [review]

File: client/web/src/site-admin/webhooks/WebhookLogNode.tsx [hunk 1]
```diff
@@ -15,13 +15,15 @@ import styles from './WebhookLogNode.module.scss'
 
 export interface Props {
     node: WebhookLogFields
+    doNotShowExternalService?: boolean
```
Comment (ryanslade) [inline client/web/src/site-admin/webhooks/WebhookLogNode.tsx:18]: nit: Maybe easier to read to reverse?

So, you have to opt in to showing external services

Comment (sashaostrikov) [reply client/web/src/site-admin/webhooks/WebhookLogNode.tsx:18]: I agree and double denial in such things is always making me feel uncomfortable. The reason I made it this way is that in fact default behavior is to show it and I didn't want to change existing usages, that's why I've added an optional param to not show ext svc.

Comment (ryanslade) [reply client/web/src/site-admin/webhooks/WebhookLogNode.tsx:18]: Cool, I guess it'll be removed completely in a few releases anyway.

**Pull request merged**